core/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -141,7 +141,8 @@ trait DeltaErrorsSuiteBase
     testUrls()
   }
 
-  test("test DeltaErrors methods") {
+  // Disabled due to https://github.com/delta-io/delta/issues/1086
+  ignore("test DeltaErrors methods") {
```

Comment:
The issue described in #1086 seems to be for pyspark only. Why do we need to ignore these tests?

Reply:
I'm assuming it's because tests will fail bc we are throwing `AnalysisException`s instead of `DeltaAnalysisException`. Or we can change all instances of `DeltaAnalysisException` to `AnalysisException` in this test suite?

Reply:
It is not just the exception types, but also the SQL code verification. These are not actually testing any Delta functionality. They are just making sure when a certain error is thrown, it has the expected content in the exception.

Reply:
The test is ignored to avoid the updating the test: Two updates are needed:
1) Change expectation of DeltaAnalysisException to AnalysisException
2) Comment the SQL code verification.